blog/2022-01-19-play-with-capact-via-cli.mdx

```diff
@@ -0,0 +1,334 @@
+
+	```bash
+	kubectl logs {PODNAME} main
+	```
+  Example output:
+
+<div className="tab-container-nested">
+<Tabs
+className="disable-tabs"
+groupId="ref"
+defaultValue="def"
+values={[
+{label: 'Default Execution', value: 'def'},
+{label: 'Policy-Changed Execution', value: 'policy'},
+]}>
+<TabItem value="def">
+
+```bash
+ ____________________________
+< message: Hello from Capact >
+ ----------------------------
+    \
+     \
+      \
+                    ##        .
+              ## ## ##       ==
+           ## ## ## ##      ===
+       /""""""""""""""""___/ ===
+  ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
+       \______ o          __/
+        \    \        __/
+          \____\______/
+```
+
+</TabItem>
+<TabItem value="policy">
+
+```bash
+ __________________________________
+< message: Never gonna give you up >
+ ----------------------------------
+    \
+     \
+      \
+                    ##        .
+              ## ## ##       ==
+           ## ## ## ##      ===
+       /""""""""""""""""___/ ===
+  ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
+       \______ o          __/
+        \    \        __/
+          \____\______/
+```
+
+</TabItem>
+</Tabs>
+</div>
+
+6. Get the Action output (TypeInstance). Copy the `id` field value:
+
+	```bash
+	capact act get hello -ojson | jq '.Actions[0].output.typeInstances'
+	```
+
+	Example output:
+	```bash {3}
+  [
+    {
+      "id": "08fcaa07-7846-47af-b6a7-7c3818c69656",
+      "typeRef": {
+        "path": "cap.type.capactio.examples.message",
+        "revision": "0.1.0"
+      }
+    }
+  ]
+	```
+
+7. Get the TypeInstance `value.message` field value:
+
+   ```bash
+   capact ti get {ID} -ojson | jq -r '.[0].latestResourceVersion.spec.value.message'
+   ```
+
+8. Delete the Action:
+
+   ```bash
+   capact act delete hello
+   ```
+
+Let's consider another example, as life isn’t all sunshine and rainbows. Sometimes we need a more positively-colored message. Capact allows you to do it with ease ✨
+
+To choose a different Implementation for the `say` Interface, we need to prepare a dedicated Policy. The power of it is that you don't have to change the main workflow. You treat the step as a building block and just swap it for another existing one.
+
+To test it, execute [steps](#steps) once again, but now select the **Policy-Changed Execution** tab.
+
+## What is so special about it?
+
+Our example was simplified to a minimum, but you could see that you can:
+
+- chain different steps together,
+- and more importantly, write once and swap implementation details with ease.
+
+Capact is also great if you want to chain different tools together and keep the same entry point (UX) but that requires more complex examples. For now, we recommend you to take a look at our other examples described in the [**How can I get involved and learn more?**](#how-can-i-get-involved-and-learn-more) section.
+
+## Behind the wall
```

Comment:
I get the reference, but are you sure it has similar meaning as "Under the hood"? Please consult that with TW or some native Enligsh speaker, if possible 🙂